packages/@aws-cdk/aws-glue/lib/table.ts

```diff
@@ -275,6 +340,11 @@ export class Table extends Resource implements ITable {
    */
   public readonly partitionIndexes?: PartitionIndex[];
 
+  /**
+   * The tables' storage descriptor properties.
+   */
+  public readonly storageDescriptor?: (StorageParameter | CustomStorageParameter)[];
```

Comment:
We can't use union types in public-facing API members

Reply:
This has been changed